app/model/admin/skillGroups.model.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var skillResource = require('../../resource/skill.resource');
+var skillGroupResource = require('../../resource/skillGroup.resource');
+var utils = require('../../utils/utils');
+
+var Promise = require('bluebird');
+
+function getSkillGroupsTemplate() {
+    return new Promise(function(resolve) {
+        var template = {
+            skills: [],
+            skillGroups: []
+        };
+        return resolve(template);
+    });
+}
+
+exports.getSkillGroupsModel = function(headers) {
+    return getSkillGroupsTemplate()
+        .then(loadSkills(headers))
+        .then(loadSkillGroups(headers));
+};
+
+function loadSkills(headers) {
+    return function(model) {
+        var skills = skillResource.getAllSkills(headers);
```

Comment:
No need to store the skills as a stand-alone variable. Just do:
 skillResource.getAllSkills(headers)
.then(function(skills){*..*}

That way you dont have to do skills.value and such.